lib/cppcheck.cpp

```diff
@@ -659,8 +659,23 @@ unsigned int CppCheck::check(const std::string &path, const std::string &content
     return checkFile(Path::simplifyPath(path), emptyString, &iss);
 }
 
+void CppCheck::fileNotFoundError(const std::string& filename)
+{
+    mErrorLogger.reportErr(ErrorMessage({ErrorMessage::FileLocation(filename, 0, 0)},
+                                        filename,
+                                        Severity::error,
+                                        "File not found",
+                                        "fileNotFound",
```

Comment:
The error might be too generic. Something more specific would be better since there might be more errors like this coming soon.